kubernetes/zenko/charts/cloudserver/templates/manager-deployment.yaml

```diff
@@ -4,15 +4,15 @@ kind: Deployment
 metadata:
   name: {{ template "cloudserver.fullname" . }}-manager
```

Comment:
Maybe we should start using partials for the different component names that a chart can have
i.e. `{{ template "cloudserver.manager.fullname" . }}

Reply:
Yeah I think that's a good idea but I maybe overkill for the cloudserver chart. I think it makes more sense for something like the backbeat chart which is expected to have many required components and references to these components. Whereas in this case this is simply an optional deployment.

Reply:
We can add it to the `TODO` list then, for when the charts are refactored.